ide/api.debugger/src/org/netbeans/api/debugger/DebuggerManager.java

```diff
@@ -785,18 +785,6 @@ public void removeDebuggerListener (DebuggerManagerListener l) {
         listeners.removeElement (l);
     }
 
-    /**
-     * Removes debugger listener.
-     * Does the same as {@link #removeDebuggerListener(org.netbeans.api.debugger.DebuggerManagerListener)},
-     * but uses the standard naming pattern, so that it can be called by
-     * {@link org.openide.util.WeakListeners}.
-     *
-     * @param l listener object.
-     */
-    // DO NOT REMOVE, used by WeakListeners, by reflection!
```

Comment:
Can you please explain why you're deleting this that says **// DO NOT REMOVE, used by WeakListeners, by reflection!** in capital letters?